app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -254,4 +256,69 @@ class SmokeTest {
             }
         }
     }
+
+    @Test
+    fun verifySearchEngineCanBeChangedTemporarilyUsingShortcuts() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        homeScreen {
+        }.openSearch {
+            verifyKeyboardVisibility()
+            clickSearchEngineButton()
+            verifySearchEngineList()
+            verifySearchEngineIconsList()
+            changeDefaultSearchEngine("Amazon.com")

```

Comment:
After this line we will need to check that the searc icon is the correct one for that search engine. It would be good to add this check:
verifySearchEngineIconsList()

after chaning the search engine each time.